raiden/api/rest.py

```diff
@@ -231,6 +231,9 @@
     ("/payments_light", PaymentLightResource),
     ("/light_client_messages", LightClientMessageResource, "Message polling"),
     ("/payments_light/create", CreatePaymentLightResource, "create_payment"),
+    (
+        '/payments_light/settlement', SettlementLightResource,
```

Comment:
I dont think this is the correct resource since settlement isnt related with payments at all. 

Take a look at `_close_light ` rest method, its invoked from `patch_channel_light` and the resource is `LightChannelsResourceByTokenAndPartnerAddress`